index.html

```diff
@@ -254,22 +254,18 @@
   <p>The application can collect, process, and act on the provided metrics via the provided JavaScript interface:</p>
 
   <pre class="example js">
-    // Gather server timing for a particular resource
-    const serverTimings = window.performance
-      .getEntriesByName("https://example.com/resource.jpg", "resource")
-      // Get just the serverTiming array
-      .map(({ serverTiming }) => serverTiming);
-
-    serverEntries
-      // we only care about "slow" ones
-      filter({ duration }  => duration > 2.0)
-      // process just the attributes we are interested in
-      .forEach(({ name, duration, description }) => {
-        // ...
-      });
-
-    // Aternatively, we convert each into JSON
-    const entriesAsJson = serverEntries.map(entry => entry.toJSON());
+    // serverTiming entries can live on 'navigation' and 'resource' entries
+    window.performance.getEntries()
+      // iterate over all entries
+      .forEach({name: url, entryType, serverTiming} => {
```

Comment:
Add a comment above explaining that `name` field is the URL of the resource, also if we're scoping down to resource then we can skip entryType and may replace it with duration (total duration)